lib/wsdl.js

```diff
@@ -2081,14 +2082,20 @@ WSDL.prototype._fromServices = function(services) {
 
 WSDL.prototype._splitQName = splitQName;
 
-WSDL.prototype._xmlnsMap = function() {
+WSDL.prototype._xmlnsMap = function (xmlnsIntoEnvelope) {
   var xmlns = this.definitions.xmlns;
   var str = '';
   for (var alias in xmlns) {
     if (alias === '' || alias === TNS_PREFIX) {
       continue;
     }
+
     var ns = xmlns[alias];
+    if (ns && xmlnsIntoEnvelope && ~xmlnsIntoEnvelope.indexOf(alias)) {
```

Comment:
do we really need the (imho too unclear) `~` operator? I'd rather go for a check of `!== -1` or use the `lodash.includes()` method (`_.includes(xmlnsIntoEnvelope)`) - just for maintaining a readable code. In addition we'd be able to drop the check for the existence of `xmlnsIntoEnvelope` while iterating

```javascript
...
if (ns && _.includes(xmlnsIntoEnvelope)) {
...
```